src/App.tsx

```diff
@@ -15,36 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABET = 'ALPHABET',
+  LENGTH = 'LENGTH',
+  '' = '',
```

Comment:
Having an empty string as a value in the `SortType` enum is not recommended. Consider using a more descriptive default value, such as `NONE` or `UNSORTED`.